app/hooks/useWalletTokens.ts

```diff
@@ -163,6 +164,24 @@ export const useWalletTokens = (walletsState: ConnectedWalletsState) => {
     };
   };
 
+  const getCosmosBalance = async (walletAddress: string) => {
+    try {
+      const client = await SigningStargateClient.connect(
+        "https://rpc.cosmos.directory/cosmoshub",
```

Comment:
Can you define this as an environment variable ? To avoid having to deploy a new version in case the public node has problems.